bower.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "md-virtual-repeater",
+  "description": "Standalone version of md-virtual-repeat directive from angular/material",
+  "main": "virtual-repeater.js",
+  "keywords": [
+    "md-virtual-repeater",
+    "angular/material",
+    "angular"
+  ],
+  "authors": [
+    "Gilles Piou <gilles.piou@gmail.com>"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/Karankang007/md-virtual-repeater"
```

Comment:
Maybe you can redirect to my repository `https://github.com/pioug/md-virtual-repeater` if you are not planning to maintain your fork